proposals/lip-0048.md

```diff
@@ -225,9 +225,8 @@ def verify(trs: Transaction) -> None:
     if trs.fee < minFee:
          raise Exception("Insufficient transaction fee")
     senderAddress = sha256(trs.senderPublicKey)[:LENGTH_ADDRESS]
-    balance = Token.getAvailableBalance(senderAddress, TOKEN_ID_FEE)
-    if trs.fee > balance:
-        raise Exception("Insufficient balance")
+    if not Token.userSubstoreExists(senderAddress, TOKEN_ID_FEE) or trs.fee > Token.getAvailableBalance(senderAddress, TOKEN_ID_FEE):
+        raise Exception("Account not initialized or insufficient balance")
```

Comment:
You're right, it would be cleaner like this:

```python
if not Token.userSubstoreExists(senderAddress, TOKEN_ID_FEE):
    raise Exception("Account not initialized")

if trs.fee > Token.getAvailableBalance(senderAddress, TOKEN_ID_FEE):
    raise Exception("Insufficient balance")
```

@gkoumout, what do you think?

Reply:
Both are fine to me, indeed the second looks a bit cleaner